src/annotator/guest.js

```diff
@@ -122,13 +122,13 @@ export default class Guest extends Delegator {
     this.element.appendChild(this.adderToolbar);
 
     this.adderCtrl = new Adder(this.adderToolbar, {
-      onAnnotate: () => {
-        this.createAnnotation();
+      onAnnotate: async () => {
+        await this.createAnnotation();
```

Comment:
This change allows the tests to check when the activity triggered by clicking "Annotate" or "Highlight" is finished.